op-challenger/metrics/metrics.go

```diff
@@ -58,9 +65,25 @@ func NewMetrics() *Metrics {
 			Name:      "up",
 			Help:      "1 if the op-challenger has finished starting up",
 		}),
+		trackedGames: *factory.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Name:      "tracked_games",
+			Help:      "Number of games being tracked by the challenger",
+		}, []string{
+			"status",
+		}),
 	}
 }
 
+const (
+	// GameStatusInProgress is the status of a game that is in progress.
+	GameStatusInProgress = "in_progress"
+	// GameStatusChallengerWon is the status of a game that the challenger won.
+	GameStatusChallengerWon = "challenger_won"
+	// GameStatusDefenderWon is the status of a game that the defender won.
+	GameStatusDefenderWon = "defender_won"
+)
```

Comment:
Label values can be pretty free form so I think we should just use `GameStatus.String()` to get the label value. Saves having to manually maintain an additional mapping of states to strings.

Reply:
I think the precedent set in the batcher's metrics are to make metricer functions explicit in what the do. Allowing any string to be passed in as a label seems like an antipattern to me since there's no input bounds. If you feel strongly that we should accept a string argument, I can change the functions to one that just accepts a string though.

Reply:
I definitely wouldn't accept an arbitrary string. I was expecting that `Metricer` would accept a `GameState` which is an enum with a fixed set of values, then it would convert that to a string using the `GameState.String()` function internally.

Reply:
I updated the game progression to return a `types.GameStatus` and pushed metrics recording into the coordinator. The issue is this tightly couples the scheduler package with internal fault types (ie the GameStatus type). What if we hoisted the GameStatus type to be a `game/` package level type as opposed to an internal fault type?

Reply:
Ah yeah, `GameStatus` is not fault dispute specific - it applies to all game types.  So makes sense to move it up to the game level.  Good spot.